src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs

```diff
@@ -38,7 +38,8 @@ public class IDEDiagnosticIDConfigurationTests
                     ValidateHelpLinkForDiagnostic(diagnosticId, descriptor.HelpLinkUri);
 
                     if (diagnosticId.StartsWith("ENC") ||
-                        !char.IsDigit(diagnosticId[^1]))
+                        !char.IsDigit(diagnosticId[^1]) ||
+                        diagnosticId == "IDE3000")
```

Comment:
can this just reference the actual named v alue in IDEDiagnosticIds?

Reply:
instead of hard coding in this value (which would have to change in two separate locations if you tweak it), just write IDEDiagnosticIds.YourNewId